gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageImpl.java

```diff
@@ -719,9 +728,12 @@ public void createEmptyObjects(
                     ioe.getClass().getSimpleName());
                 logger.atFine().withCause(ioe).log("Ignored exception while creating empty object");
               } else {
+                hasException.set(true);
```

Comment:
So we will count this only once?

Reply:
Yes. This is one way of doing it. I saw that we are already having "innerExceptions" storing exception so we may not need "hasException" variable even. Updated it.